BeagleBone_black/src/motor_control/controller.cpp

```diff
@@ -185,8 +213,8 @@ void Controller::configure()
   SDOMessage.data[6]   = 0x01;
   SDOMessage.data[7]   = 0x00;
 
-  sendSdoCan(SDOMessage);
   log_.DBG1("MOTOR", "Controller %d: Configuring motor rated current", node_id_);
+  sendSDO(SDOMessage);
 
   // Set motor rated torque to 80N
```

Comment:
Isn't [torque](https://en.wikipedia.org/wiki/Torque) measured in Nm rather than N?